tox.ini

```diff
@@ -24,6 +24,8 @@ format = pylint
 [testenv:syntax]
 deps = 
     flake8
+    # NOTE(retr0h): We specifically pin to a known version so formatting
+    # is consistent across developer/PR.
     yapf==0.10.0
```

Comment:
I absolutely agree with pinning the runtime requirements, but we should not put packages likes that in tox. If we do that, people cannot know that they need to install yapf 0.10.0.